.gitignore

```diff
@@ -17,6 +17,3 @@ venv.bak/
 
 # Logs
 *.log
-
-/node_modules
```

Comment:
Removing this since `node_modules` now exists in the `client` directory.

Reply:
Correct me if I'm wrong but I think this syntax means all folders regardless of location named node_modules gets ignored (like the __pycache__) entry. So I think that we keep the node_modules line

Reply:
Ohhh I see now what you did, I see the new gitignore in client, ignore my previous comment

Yes we should remove the node_modules line